cmd/tapRunner.go

```diff
@@ -106,6 +106,9 @@ func tap() {
 	go watchHubEvents(ctx, kubernetesProvider, cancel)
 	go watchHubPod(ctx, kubernetesProvider, cancel)
 	go watchFrontPod(ctx, kubernetesProvider, cancel)
+	if !config.Config.Tap.NoTelemetry {
+		log.Info().Msg("Starting Telemetry...")
+	}
```

Comment:
```suggestion
	log.Info().Bool("enabled", !config.Config.Tap.NoTelemetry).Msg("Telemetry")
```

Also print it before this line;

https://github.com/kubeshark/kubeshark/blob/7afb1d8b9b16011431da95e05dfcbee749d33bd8/cmd/tapRunner.go#L80

Reply:
done